diff --git a/configure.ac b/configure.ac
index 4f9fb38..6703b5d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -457,11 +457,12 @@ CheckJACK()
 AC_HELP_STRING([--enable-jack], [support the JACK audio API [[default=yes]]]),
                   , enable_jack=yes)
     if test x$enable_jack = xyes; then
-        PKG_CHECK_MODULES(JACK, jack, [have_jack=yes], [have_jack=no])
-
+	have_jack=yes
         if test x$have_jack = xyes; then
                 AM_CONDITIONAL(HAVE_JACK, true)
                 AC_DEFINE([HAVE_JACK], [1], [Define if we are compiling with JACK support.])
+                LIBS="$LIBS `pkg-config --libs jack`"
+                CPPFLAGS="$CPPFLAGS `pkg-config --cflags jack`"
         fi
     fi
 
@@ -571,8 +572,9 @@ else
 	AC_DEFINE([PSS_STYLE],[1], [Defines the filesystem path-separator type.])
 
 	if test x$with_libsndfile = xyes; then
-		PKG_CHECK_MODULES(SNDFILE, sndfile >= 1.0.2, [], AC_MSG_ERROR([*** libsndfile >= 1.0.2 not found!]))
-                AC_DEFINE([HAVE_LIBSNDFILE], [1], [Define if we are compiling with libsndfile support.])
+                SNDFILE_LIBS="-lsndfile"
+                SNDFILE_CFLAGS=""
+		AC_DEFINE([HAVE_LIBSNDFILE], [1], [Define if we are compiling with libsndfile support.])
 		AM_CONDITIONAL(HAVE_LIBSNDFILE, true)
 	fi
 
